doc/book/replication/repl_sync.rst

```diff
@@ -35,9 +35,16 @@ To control the behavior of synchronous transactions, there exist global
 
     box.cfg{replication_synchro_quorum = <number of instances>}
 
-This option tells how many replicas should confirm the receipt of a synchronous
-transaction before it is committed. Since version :doc:`2.10.0 </release/2.10.0>`,
-this option does not account for anonymous replicas. As a usage example, consider this:
+..  code-block:: lua
+
+    box.cfg{replication_synchro_quorum = "(cluster_size/2)+1"}
```

Comment:
This example won't work. Better say
`box.cfg{replication_synchro_quorum = "N / 2 + 1"}`
like in other places.